src/components/Footer/css/footer.css

```diff
@@ -0,0 +1,18 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+footer {
+    position: absolute;
+    width: 100%;
+    height: 97px;
+    left: 0px;
+    top: 609px;
```

Comment:
@BarnaTB 
Won't specifying the position become problematic when the body gets populated and is more than 609px in height.
What do you think?

Reply:
@Opio-Emmanuel-Omona I gave it height this time because if I don't it'd be mixed up with the nav bar but I could optionally give it margin-top. Hopefully, it won't act up. Thanks.